pages/profile-page/profile-page.html

```diff
@@ -0,0 +1,181 @@
+                            <div
+                                class="modal-header flex justify-between align-items-center bg-grey-800 text-white p-5">
+                                <h4 class="h4 text-lg">User Info</h4>
+                                <span onclick="closeModal()"
+                                    class="modal-close text-rose-500 text-hover-rose-300">×</span>
+                            </div>
+                            <div class="modal-body p-5">
+                                edit info in here
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+            </div>
+        </div>
+
+    </main>
+    <!-- main -->
+
+    <!-- footer -->
+    <footer style="bottom: 0"
+        class="relative flex flex-col h-44 justify-center align-items-center w-full p-5 z-50 bg-fuchsia-100">
+        <div class="text-grey-700">
+            Made with
+            <span class="text-fuchsia-700 font-bold ml-1 mr-1">&lt;/&gt; </span> by
+            <a href="https://github.com/VedanthB" style="padding: 0; text-transform: none"
+                class="ml-1 btn btn-link-fuchsia">vedanth</a>
+        </div>
+        <div class="flex mt-3 align-items-center justify-center">
+            <a href="https://github.com/VedanthB">
+                <i class="fab fa-github mr-2 text-fuchsia-700"></i>
+            </a>
+            <a href="https://twitter.com/vedanth_X0X0">
+                <i class="fab fa-twitter mr-2 text-fuchsia-700"></i>
+            </a>
+            <a href="https://www.linkedin.com/in/vedanth-bora/">
+                <i class="fab fa-linkedin mr-2 text-fuchsia-700"></i>
+            </a>
+            <a href="https://dev.to/vedanthb">
+                <i class="fab fa-dev text-fuchsia-700"></i>
+            </a>
+        </div>
+        <div class="text-grey-700 mt-3">
+            Designed using
+            <a href="https://argon-css.netlify.app/" style="padding: 0; text-transform: none"
+                class="ml-1 btn btn-link-fuchsia">Argon CSS</a>
+        </div>
+
+    </footer>
+    <!-- footer -->
+
+    <script>
+        // / Get the modal
+        var modal = document.getElementById("modal");
+
+        // Get the button that opens the modal
+        var btn = document.getElementById("modalBtn");
+
+        // // Get the  element that closes the modal
+        var span = document.querySelector("modal-close");
+
+        // When the user clicks the button, open the modal
+        function openModal() {
+            modal.style.display = "block";
+        }
+
+        // When the user clicks on  (x), close the modal
+        function closeModal() {
+            modal.style.display = "none";
+        }
+
+        // When the user clicks anywhere outside of the modal, close it
+        window.onclick = function (event) {
+            if (event.target == modal) {
+                modal.style.display = "none";
```

Comment:
Instead of using js in the Html file create separate .js file and use it. no need to have comments on each line